### ISSUE TITLE
Option to save exact plugin version to config.xml

### ISSUE THREAD

**Issue by biesbjerg** [dpa99c/cordova-check-plugins#20]
When updating plugins, plugin version in config.xml is always set to ~VERSION. 

Is it possible to save exact version?

**Comment by dpa99c** [comment]
TL;DR: Without this tool taking over versioning from the Cordova CLI, it's not possible to save the exact version without the `~`.

The saving of plugin versions is actually being undertaken by the Cordova (or Phonegap) CLI - if the `--save` argument is present, this tool [forwards it onto the Cordova CLI command](https://github.com/dpa99c/cordova-check-plugins/blob/master/lib/update.js#L91). 

So `cordova-check-plugins --update=cordova-plugin-device --save` is the same as doing `cordova plugin rm cordova-plugin-device && cordova add cordova-plugin-device --save`.

Therefore the `~` added to the version number is done by the Cordova CLI. This was changed from '^' some time ago ([see here](https://github.com/apache/cordova-lib/pull/248/files)).

**Comment by kelvindart** [comment]
They now support this tag - is there a way to include it in the plugin?
